brands/shop/supermarket.json

```diff
@@ -2364,6 +2376,16 @@
       "name": "La Vie Claire",
       "organic": "only",
       "shop": "supermarket"
+      }
+    },
+      "shop/supermarket|La Sirena": {
+    "countryCodes": ["es"],
+    "tags": {
+      "brand": "La Sirena",
+      "brand:wikidata": "Q21034458",
+      "brand:wikipedia": "ca:La_Sirena",
+      "name": "La sirena",
+      "shop": "frozen_food"
```

Comment:
Our build script issued a warning because of this `shop=frozen_food` tag for an entry in the `shop/supermarket.json` file.  

In b173fb21 I switched the tag to `shop=supermarket` but let me know if it should be frozen food.  Also I fixed the name to have a capital 'S'.